ProjectSettings/DynamicsManager.asset

```diff
@@ -12,4 +12,4 @@ PhysicsManager:
   m_SolverIterationCount: 6
   m_QueriesHitTriggers: 1
   m_EnableAdaptiveForce: 0
-  m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```

Comment:
I don't love this one, however as things are I need physics hands and I need them to not collide with interaction objects.  My thinking was that this was OK for the demo, and that we have to document this requirement.